### ISSUE TITLE
Unify methods of showing help

### ISSUE THREAD

**Issue by jeanas** [wbsoft/frescobaldi#1624]
While working on the user guide, I noticed that all these methods of providing help to the user are used in Frescobaldi:

- Making some text show on hover,
- The "Help > What's This?" menu item,
- Some "Help" buttons (e.g., in the preferences),
- Contextual menu (e.g., in the music view).

Perhaps we should be more consistent?

**Comment by PeterBjuhr** [comment]
Agreed, it could be more consistent. But I don't mind for example that dialogs have a help button and widgets have a contextual help. The "What's This?" functionality also seems to be an own thing.
